bnd.bnd

```diff
@@ -0,0 +1,32 @@
+Bundle-Activator: \
+    ${classes;IMPLEMENTS;org.osgi.framework.BundleActivator}
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
+
+Export-Package:\
+    org.apache.sling.junit.annotations;version=1.1.0,\
```

Comment:
The default is to export all packages with a version annotation in that version. Let's stick to that default.

Reply:
Fixed in the latest changes. Will do another pull request.